src/Four_Player_Adapter.md

```diff
@@ -5,41 +5,136 @@ to connect for multiplayer via [serial data transfer](./Serial_Data_Transfer_(Li
-3 "STAT" bytes are sent indicating the current connection status of the other
-Game Boys. Each byte is usually the same, however, sometimes the status can
-change midway through a ping, typically on STAT2 or STAT3. Each STAT byte
-looks like such:
+The chart below illustrates how Game Boys should respond to bytes in a ping packet.
+- Note: When a byte in the DMG-07 column is received the matching byte in the Reply
+column should be loaded into the [SB] register as a reply that will be transmitted
+during the next serial transfer.
+
+Received From DMG-07 | Game Boy reply sent during next transfer
+---------------------|-----------------------
+PING HEADER (\$FE)   | ACK1 = (\$88)
+STAT1                | ACK2 = (\$88 )
+STAT2                | RATE (Packet Timing)
+STAT3                | SIZE (Packet Size)
+
+### ACK Replies
+
+The ACK1 and ACK2 bytes use a fixed value (\$88) and are sent in reply to
+the PING HEADER and STAT1 bytes.
+
+### RATE Setting
+The RATE setting configures packet timing and works differently in the ping and
+transmission phases. It is sent in reply to the STAT2 byte.
+
+Note: In both phases the value \$00 for RATE has special behavior where it does not
+change the speed, so it should not be used.
+
+#### RATE in Ping Phase
+In ping phase RATE only adjusts the delay between packets (byte timing is unchanged)
+and takes effect immediately upon the next packet. The timing is calculated as follows:
+```
+Delay between packets = (12.2 milliseconds) + ((RATE & 0x0F) * 1 millisecond)
+```
+Where:
+- Transfer time for all bytes: 4.71 milliseconds (always)
+- Delay between packets: 12.2 to 27.21 milliseconds
+
+This yields a range of 12.20 to 27.21 milliseconds for the total packet and delay time.
+
+#### RATE in Transmission Phase
+In transmission phase the Clock Rate setting is determined by the **last** RATE value
+transmitted before exiting ping phase. The timing is more complex and is calculated
+as described below.
+
+First determine the delay between packet bytes:
+```
+Delay between bytes = ((RATE >> 4) x .106 milliseconds) + 0.887 milliseconds
+```
+
+Then the total packet and delay time will be whichever of the following is larger:
+```
+((RATE & 0x0F) x 1 milliseconds) + 17 milliseconds
+or
+(Transfer time per byte + Delay between bytes) x Byte Count) + (between .36 to 2.15 milliseconds)
+```
+Where:
+- Transfer time per byte: ~0.128 microseconds
+- Byte count: 4, 8, 12 or 16 depending on the setting for SIZE
+- The formula for calculating the .36 to 2.15 milliseconds add-on is not yet understood.
+  
+This yields a range of 17.0 to 41.6 milliseconds for the total packet and delay time.
+
+### SIZE Packet Size setting
+SIZE sets the number of usable data bytes sent by each Game Boy during a packet
+in transmission phase. It is sent in reply to the STAT3 byte.
 
+The total number of bytes broadcasted by the DMG-07 in a packet will be SIZE x 4.
+For example if SIZE is 3 then the total packet size will be 12 bytes (3 x 4). 
```

Comment:
`x` is for variables (see `git grep -w x '*.md'`), `×` is for multiplication (see `git grep '×'`).
```suggestion
The total number of bytes broadcasted by the DMG-07 in a packet will be SIZE×4.
For example if SIZE is 3 then the total packet size will be 12 bytes (3×4). 
```